### ISSUE TITLE
Adjust image preview support test

### ISSUE THREAD

**Issue by Toflar** [contao/contao#5178]
### Affected version(s)

4.13.*, 5.0.*

### Description

See https://github.com/contao/contao/blob/5.x/core-bundle/src/Image/Preview/ImaginePreviewProvider.php#L103

**Comment by leofeyer** [comment]
I have tested if the changes work but `$this->imagine->getDriverInfo()->isFormatSupported('pdf')` kept returning `false` whereas `\in_array('PDF'), \Imagick::queryFormats('PDF'))` returned `true`.

<img width="334" alt="33" src="https://user-images.githubusercontent.com/1192057/187191368-74d5ab76-0dec-4fd0-b6af-dc86ff710cb1.png">

So I guess this does not work yet.

**Comment by ausi** [comment]
Your screenshot suggests, that `$this->imagine` was using `GD` while `Imagick::queryFormats` uses imagemagick.

But from reading the Imagine source code, it looks like the supported formats would never include PDF as it is based on a fixed list of formats: https://github.com/php-imagine/Imagine/blob/9b9aacbffadce8f19abeb992b8d8d3a90cc2a52a/src/Image/Format.php#L14-L32

So I think we cannot use `->getDriverInfo()->isFormatSupported()` here. The difference is probably because we want to know which formats imagemagick supports for *reading* but the driver info only lists formats that can be written.

**Comment by leofeyer** [comment]
So we can remove these lines again I guess:

https://github.com/contao/contao/blob/9016d5656034168030cf1c2997fd9a3c3f91260b/core-bundle/src/Image/Preview/ImaginePreviewProvider.php#L103-L107

**Comment by ausi** [comment]
Yes, they can be removed, but maybe added to the format detection in the image size DCA: https://github.com/contao/contao/blob/652aebc61409c0edd4014ddfd1f5cc2536926418/core-bundle/contao/dca/tl_image_size.php#L424
